src/js/translations/en-US.json

```diff
@@ -8,5 +14,24 @@
   "DASHBOARD.PANEL_HEADING.COMPONENT_HEALTH": "Component Health",
   "DASHBOARD.PANEL_HEADING.NODES": "Nodes",
 
-  "SERVICE_ACTIONS.OPEN_SERVICE": "Open Service"
+  "DOCS.MORE_INFORMATION": "More information",
+  "DOCS.DOCUMENTATION": "documentation",
+
+  "SERVICE_ACTIONS.DELETE": "Delete",
```

Comment:
I believe we could use some of these in `COMMON` instead since it's very generic. Not big deal though.